testing/console-logging-setup.js

```diff
@@ -63,22 +79,12 @@ function printWarning(...messages) {
   const message = messages.join(' ');
 
   // Match equal strings.
```

Comment:
Nit: For documentation, worth moving this comment and the one on line 71 to `indexOfExpectedMessage()`?

Reply:
Done.